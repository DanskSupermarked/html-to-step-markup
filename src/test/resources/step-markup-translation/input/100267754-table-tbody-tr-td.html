<p>Olivers Fisk Classic til katte er et sundt og velsmagende, komplet og afbalanceret kvalitetsfoder til voksne katte, som elsker fisk.Olivers Fisk Classic har et højt indhold af laks og et afbalanceret indhold af omega-3 og omega-6 fedtsyrer. Olivers Fisk Classic er formuleret ifølge Olivers høje kvalitetskrav, og indeholder ud over den korrekte balance af alle de vigtige næringsstoffer, som katte har behov for hver eneste dag, yderligere næringsstoffer, som Olivers mener gavner din kats sundhed. Foderet er derfor optimeret med fiskeolie og funktionelle fibre.</p> <p>Et kostskifte kan være voldsomt for kattens mave. Derfor anbefaler vi, at katten gradvist over 3-7 dage vænner sig til sig vid det nye foder ved at blande det nye foder med det gamle. I de første dage med størst andel af det gamle foder og til slut omvendt.</p> <p>Olivers Fisk Classic er fremstillet af de bedste ingredienser af høj kvalitet. Hos Olivers tilsætter vi naturligvis ingen kunstige farvestoffer, smagsstoffer eller konserveringsmidler til vores foder.</p><p><b>Fodervejledning:</b></p> <table width="254" height="238" cellspacing="1" cellpadding="1" border="1">     <tbody>         <tr>             <td><b>Kattens vægt (kg)</b></td>             <td><b>Foder (gram pr. dag)</b></td>         </tr>         <tr>             <td>&nbsp;2</td>             <td>&nbsp;40</td>         </tr>         <tr>             <td>&nbsp;3</td>             <td>&nbsp;55</td>         </tr>         <tr>             <td>&nbsp;4</td>             <td>&nbsp;65</td>         </tr>         <tr>             <td>&nbsp;5</td>             <td>&nbsp;75</td>         </tr>         <tr>             <td>&nbsp;6</td>             <td>&nbsp;85</td>         </tr>         <tr>             <td>&nbsp;7</td>             <td>&nbsp;95</td>         </tr>         <tr>             <td>&nbsp;8</td>             <td>&nbsp;105</td>         </tr>         <tr>             <td>&nbsp;9</td>             <td>&nbsp;110</td>         </tr>         <tr>             <td>&nbsp;10</td>             <td>&nbsp;120</td>         </tr>     </tbody> </table>
