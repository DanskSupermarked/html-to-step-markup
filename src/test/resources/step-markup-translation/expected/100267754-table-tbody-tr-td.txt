Olivers Fisk Classic til katte er et sundt og velsmagende, komplet og afbalanceret kvalitetsfoder til voksne katte, som elsker fisk.Olivers Fisk Classic har et højt indhold af laks og et afbalanceret indhold af omega-3 og omega-6 fedtsyrer. Olivers Fisk Classic er formuleret ifølge Olivers høje kvalitetskrav, og indeholder ud over den korrekte balance af alle de vigtige næringsstoffer, som katte har behov for hver eneste dag, yderligere næringsstoffer, som Olivers mener gavner din kats sundhed. Foderet er derfor optimeret med fiskeolie og funktionelle fibre.<return/>
<return/>
Et kostskifte kan være voldsomt for kattens mave. Derfor anbefaler vi, at katten gradvist over 3-7 dage vænner sig til sig vid det nye foder ved at blande det nye foder med det gamle. I de første dage med størst andel af det gamle foder og til slut omvendt.<return/>
<return/>
Olivers Fisk Classic er fremstillet af de bedste ingredienser af høj kvalitet. Hos Olivers tilsætter vi naturligvis ingen kunstige farvestoffer, smagsstoffer eller konserveringsmidler til vores foder.<return/>
<return/>
<bold>Fodervejledning:</bold> <return/>
<bold>Kattens vægt (kg)</bold> | <bold>Foder (gram pr. dag)</bold><return/>
2 | 40<return/>
3 | 55<return/>
4 | 65<return/>
5 | 75<return/>
6 | 85<return/>
7 | 95<return/>
8 | 105<return/>
9 | 110<return/>
10 | 120
